org.contour.Terminal.yml

```diff
@@ -0,0 +1,92 @@
+app-id: org.contour.Terminal
+name: Contour Terminal Emulator
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: contour
+finish-args:
+  - --filesystem=host
+  - --device=all          # we need access to at least DRI (for OpenGL) & PTY
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --own-name=org.contour.Terminal
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
why do you need this?

Reply:
Because of sandbox privilege escalation. It's a terminal and I need to [escape from the sandbox](https://github.com/contour-terminal/contour/blob/4f565357720915402aca5ca63ca1fa812ed4741f/src/terminal/Process_unix.cpp#L170-L191).